Resources/Prototypes/Entities/Objects/Weapons/Melee/pickaxe.yml

```
@@ -73,7 +73,7 @@
     heavyRateModifier: 1
     heavyRangeModifier: 2
     heavyDamageBaseModifier: 1
-    heavyStaminaCost: 2
+   # heavyStaminaCost: 2 Floof change
```

Default is 2.5 per hit.
```suggestion
   heavyStaminaCost: 0 # Floof change
```